libraries/chain/db_block.cpp

```diff
@@ -624,8 +624,7 @@ processed_transaction database::_apply_transaction(const signed_transaction& trx
 { try {
    uint32_t skip = get_node_properties().skip_flags;
 
-   if( true || !(skip&skip_validate) )   /* issue #505 explains why this skip_flag is disabled */
-      trx.validate();
+   trx.validate();
```

Comment:
As described in https://github.com/cryptonomex/graphene/issues/505, the disabling of `skip_validate` was temporary, although it had never worked. Perhaps we can introduce it again at some time in the future to improve performance? Another note: `validate()` is not only called here, maybe we'll need to change other code (if to skip validation completely).

Reply:
AFAICS BM introduced the flag to get a stuck chain going again. I couldn't find a reasonable explanation in cryptonomex issue 505.

I think the flag doesn't make much sense. In the future, trx.validate will be called before the transaction is handed over to the database. trx.validate() then only checks the preprocessed _validate flag (which will always be true because otherwise the preprocessing would have failed, at least that's the theory).